default/python/AI/AIDependencies.py

```diff
@@ -299,10 +301,6 @@ class Tags:
 
 EXOBOT_TECH_NAME = "PRO_EXOBOTS"
 
-SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
-SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
-SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
```

Comment:
How do these changes relate to this PR (which is about rebalancing flat bonus techs)?

Reply:
they don't relate to rebalance, they are grooming. Should they be removed from the PR?

Reply:
I don't have a particular strong opinion on the matter, but generally I think it's better not to include such grooming stuff in PRs if it's not related in any way. Such things can be committed directly, without putting them into PRs.

IMO it is potentially confusing for reviewers when there are completely unrelated changes here and there in a PR...

No need to remove these changes now from this PR, but try to avoid it in the future.